packages/components/stories/timeline/timeline.stories.js

```diff
@@ -4,26 +4,31 @@ import { boolean, withKnobs } from '@storybook/addon-knobs';
 import { withInfo } from '@storybook/addon-info';
 import Timeline from 'Components/timeline';
 import Theme from '../shared/theme';
+import notes from './README.md';
 
 const stories = storiesOf('Timeline', module);
 
 stories.addDecorator(withKnobs).addDecorator(withInfo);
 
-stories.add('basic usage', () => (
-    <Theme is_dark={boolean('Theme', true)}>
-        <div style={{ display: 'flex' }}>
-            <div>left component</div>
-            <Timeline>
-                <Timeline.Item item_title='Title'>
-                    <div>test</div>
-                </Timeline.Item>
-                <Timeline.Item item_title='Title2'>
-                    <div>test2</div>
-                </Timeline.Item>
-                <Timeline.Item item_title='Title3'>
-                    <div>test3</div>
-                </Timeline.Item>
-            </Timeline>
-        </div>
-    </Theme>
-));
+stories.add(
```

Comment:
@fatemeh-fs Please don’t add unnecessary changes to the code like here. Specially on deriv-app project main files. Here for storybooks it’s not a big problem but for main file this changes will make it hard to check all changes you made.
Thanks